exercises/practice/list-ops/.meta/config.json

```diff
@@ -1,6 +1,53 @@
 {
   "blurb": "Implement basic list operations",
   "authors": [],
```

Comment:
Another empty one, added in https://github.com/exercism/elixir/commit/7e23cf93269ca6a4b31176b9fbd79918253387f3#diff-639ed1e0b89310f36410cd8377faf89a9d2c9ecc438fbb15b2657b237aefdcc1